javatests/arcs/android/host/AndroidEntityHandleManagerTest.kt

```diff
@@ -0,0 +1,252 @@
+package arcs.android.host
+
+import android.app.Application
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.work.testing.WorkManagerTestInitHelper
+import arcs.android.storage.handle.AndroidHandleManager
+import arcs.core.data.FieldType
+import arcs.core.data.Schema
+import arcs.core.data.SchemaFields
+import arcs.core.data.SchemaName
+import arcs.core.host.HandleMode
+import arcs.core.host.EntityHandleManager
+import arcs.core.host.HandleHolder
+import arcs.core.storage.driver.RamDiskStorageKey
+import arcs.core.storage.referencemode.ReferenceModeStorageKey
+import arcs.core.testutil.assertThrows
+import arcs.sdk.ReadWriteCollection
+import arcs.sdk.ReadWriteSingleton
+import arcs.sdk.ReadableCollection
+import arcs.sdk.ReadableSingleton
+import arcs.sdk.WritableCollection
+import arcs.sdk.WritableSingleton
+import arcs.sdk.android.storage.service.DefaultConnectionFactory
+import arcs.sdk.android.storage.service.testutil.TestBindingDelegate
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.TestCoroutineScope
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.coroutines.experimental.suspendCoroutine
+
+typealias Person = TestParticleInternal1
+
+@Suppress("EXPERIMENTAL_API_USAGE")
+@RunWith(AndroidJUnit4::class)
+class AndroidEntityHandleManagerTest {
+    private lateinit var app: Application
+
+    val entity1 = Person("Jason", 21.0, false)
+    val entity2 = Person("Jason", 22.0, true)
+    lateinit var handleHolder: TestParticleHandles
+
+    private val schema = Schema(
+        listOf(SchemaName("Person")), SchemaFields(
+            singletons = mapOf(
+                "name" to FieldType.Text, "age" to FieldType.Number, "is_cool" to FieldType.Boolean
+            ), collections = emptyMap()
+        ), "1234acf"
+    )
+
+    private val singletonKey = ReferenceModeStorageKey(
+        backingKey = RamDiskStorageKey("single-back"), storageKey = RamDiskStorageKey("single-ent")
+    )
+
+    private val setKey = ReferenceModeStorageKey(
+        backingKey = RamDiskStorageKey("set-back"), storageKey = RamDiskStorageKey("set-ent")
+    )
+
+    @Before
+    fun setUp() {
+        app = ApplicationProvider.getApplicationContext()
+        app.setTheme(R.style.Theme_AppCompat);
+
+        // Initialize WorkManager for instrumentation tests.
+        WorkManagerTestInitHelper.initializeTestWorkManager(app)
+
+        handleHolder = TestParticleHandles()
+    }
+
+    fun handleManagerTest(block: suspend TestCoroutineScope.(EntityHandleManager) -> Unit) =
+        runBlockingTest {
```

Comment:
If you put the arg on its own line, then the `=` and the `runBlockingTest` can go on the same line, and everything can be indented less:

```
fun handleManagerTest(
    block: suspend TestCoroutineScope.(EntityHandleManager) -> Unit
) = runBlockingTest {
    ...
}
```